deploy/nightly-bundle/deploy.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+set -ex
+
+RED='\033[0;31m'
+NO_COLOR='\033[0m'
+
+globalNamespace=`oc -n openshift-operator-lifecycle-manager get deployments catalog-operator -o jsonpath='{.spec.template.spec.containers[].args[1]}'`
+echo "Global Namespace: ${globalNamespace}"
+
+PACKAGE="${PACKAGE:-kubevirt-hyperconverged}"
+CS_SOURCE="${CS_SOURCE:-hco-catalogsource}"
+
+TARGET_NAMESPACE="${TARGET_NAMESPACE:-kubevirt-hyperconverged}"
+MARKETPLACE_NAMESPACE="${MARKETPLACE_NAMESPACE:-openshift-marketplace}"
+HCO_REGISTRY_IMAGE="${HCO_REGISTRY_IMAGE:-quay.io/kubevirt/hco-container-registry:latest}"
+POD_TIMEOUT="${POD_TIMEOUT:-360s}"
+HCO_VERSION="${HCO_VERSION:-1.0.0}"
+HCO_CHANNEL="${HCO_CHANNEL:-1.0.0}"
```

Comment:
Please use `1.1.0`

Reply:
@nunnatsa  this is a hard link to `deploy_marketplace.sh`, not a new file so if possible, I'd rather not changing it right now...